cvat-ui/src/reducers/annotation-reducer.ts

```diff
@@ -437,6 +437,30 @@ export default (state = defaultState, action: AnyAction): AnnotationState => {
                 },
             };
         }
+        case AnnotationActionTypes.ADD_TAG: {
+            const {
+                labelID,
+                objectType,
+                activeControl,
+            } = action.payload;
+
+            return {
+                ...state,
+                annotations: {
+                    ...state.annotations,
+                    activatedStateID: null,
+                },
+                canvas: {
+                    ...state.canvas,
+                    activeControl,
+                },
+                drawing: {
+                    ...defaultState.drawing,
+                    activeLabelID: labelID,
+                    activeObjectType: objectType,
+                },
+            };
+        }
```

Comment:
Why do not use ``AnnotationActionTypes.DRAW_SHAPE``? Actually it does the same as far as I understood.
We can rename it for better understanding (for example: ``REMEMBER_CREATE_PARAMETERS``)